src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,48 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  let internalActiveTab = tabs.find(tab => tab.id === activeTabId);
+
+  let internalActiveTabId = activeTabId;
+
+  if (!internalActiveTab) {
+    const [firstTab] = tabs;
+
+    internalActiveTab = firstTab;
+    internalActiveTabId = firstTab.id;
+  }
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={classNames({
+                'is-active': internalActiveTabId === tab.id,
+              })}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (internalActiveTabId !== tab.id) {
+                    onTabSelected(tab.id);
+                  }
+                }}
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here